build/WinUI-OB-Official.yml

```diff
@@ -182,6 +186,68 @@ extends:
             displayName: 'list contents of ArtifactStagingDirectory'
             continueOnError: true
 
+    - stage: symbols
+      dependsOn: build
+      jobs:
+      - job: publishsymbols
+        pool:
+          type: windows  
+        variables:
+          ob_outputDirectory: $(build.artifactStagingDirectory)
+          pdbDir: $(Build.SourcesDirectory)\Artifacts\drop
+        steps:
+        - task: DownloadPipelineArtifact@2
+          inputs:
+            artifactName: drop_build_main_x64_Release
+            targetPath: $(pdbDir)
+            buildType: specific
+            buildVersionToDownload: specific
+            project: $(System.TeamProjectId)
+            definition: $(System.DefinitionId)
+            pipelineId : $(Build.BuildId)
+            itemPattern: '**/*.pdb'
+
+        - task: DownloadPipelineArtifact@2
+          inputs:
+            artifactName: drop_build_main_x86_Release
+            targetPath: $(pdbDir)
+            buildType: specific
+            buildVersionToDownload: specific
+            project: $(System.TeamProjectId)
+            definition: $(System.DefinitionId)
+            pipelineId : $(Build.BuildId)
+            itemPattern: '**/*.pdb'
+
+        - task: DownloadPipelineArtifact@2
+          inputs:
+            artifactName: drop_build_main_arm_Release
+            targetPath: $(pdbDir)
+            buildType: specific
+            buildVersionToDownload: specific
+            project: $(System.TeamProjectId)
+            definition: $(System.DefinitionId)
+            pipelineId : $(Build.BuildId)
+            itemPattern: '**/*.pdb'
+
+        - task: DownloadPipelineArtifact@2
+          inputs:
+            artifactName: drop_build_main_arm64_Release
+            targetPath: $(pdbDir)
+            buildType: specific
+            buildVersionToDownload: specific
+            project: $(System.TeamProjectId)
+            definition: $(System.DefinitionId)
+            pipelineId : $(Build.BuildId)
+            itemPattern: '**/*.pdb'
+
+        - script: dir /s /b $(pdbDir)
+          displayName: Dump symbols staging directory
+
+        - template: ${{variables['System.DefaultWorkingDirectory']}}\AzurePipelinesTemplates\WindowsAppSDK-PublishSymbol-Steps.yml@WindowsAppSDKConfig
+          parameters:
+            SearchPattern: '$(pdbDir)\**\*.pdb'
+            IsOfficial: true
```

Comment:
FYI just so you are aware
IsOfficial: true
controls whether we publish publicly. Maybe there should be a more obvious name for it.

Reply:
Sounds good. We use WinUI-OB-Official for official builds, so it makes sense for it to publish the symbols.